src/sidebars/search/SearchSideBar.vue

```diff
@@ -11,96 +11,93 @@ import { searchStore } from '../../store/store.js'
 			<template #icon>
 				<Magnify :size="20" />
 			</template>
-			Zoek snel in het voor uw beschikbare federatieve netwerk<br>
-			<NcTextField class="searchField"
-				:value.sync="searchStore.search"
-				label="Zoeken" />
-			<NcNoteCard v-if="searchStore.searchError" type="error">
-				<p>{{ searchStore.searchError }}</p>
-			</NcNoteCard>
+			<NcSelect v-bind="registerOptions" v-model="selectedRegister" input-label="Registratie" />
+			<NcSelect v-bind="schemaOptions" v-model="selectedSchema" input-label="Schema" />
```

Comment:
Could you add a loading state to registerOptions.
And could you make the schemaOptions disabled if there is no register selected.